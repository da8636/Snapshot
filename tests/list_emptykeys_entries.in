LIST KEYS
SET a 1 2
SET b 3 4
LIST ENTRIES
BYE
