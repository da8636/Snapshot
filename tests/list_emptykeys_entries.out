no keys

ok

ok

b [3 4]
a [1 2]

bye
